doc/contributing.rst

```diff
@@ -362,14 +391,15 @@ Code Formatting
 xarray uses several tools to ensure a consistent code format throughout the project:
 
 - `Black <https://black.readthedocs.io/en/stable/>`_ for standardized
-  code formatting
+  code formatting,
 - `blackdoc <https://blackdoc.readthedocs.io/en/stable/>`_ for
-  standardized code formatting in documentation
-- `Flake8 <http://flake8.pycqa.org/en/latest/>`_ for general code quality
-- `isort <https://github.com/timothycrosley/isort>`_ for standardized order in imports.
-  See also `flake8-isort <https://github.com/gforcada/flake8-isort>`_.
+  standardized code formatting in documentation,
+- `flake8 <http://flake8.pycqa.org/en/latest/>`_ for general code quality (we now use `ruff <https://github.com/charliermarsh/ruff/>`_ to apply these checks),
+- `isort <https://github.com/timothycrosley/isort>`_ for standardized order in imports
+  (again we use `ruff <https://github.com/charliermarsh/ruff/>`_ to apply this formatting),
+- `absolufy-imports <https://github.com/MarcoGorelli/absolufy-imports>`_ for absolute instead of relative imports from different files,
```

Comment:
```suggestion
- `ruff <https://github.com/charliermarsh/ruff/>`_ for code quality checks and standardized order in imports
- `absolufy-imports <https://github.com/MarcoGorelli/absolufy-imports>`_ for absolute instead of relative imports from different files,
```